src/app/static/src/app/types.ts

```diff
@@ -182,22 +182,29 @@ export interface SelectedADRUploadFiles {
 }
 
 export interface GenericChartTableConfig {
-    columns: {
-        data: {
-            columnId: string,
-            labelFilterId: string | null
-        },
-        header: {
-            type: "filterLabel" | "selectedFilterOption",
-            filterId: string
-        }
-    }[]
+    columns: GenericChartTableColumnConfig[]
+}
+
+export interface GenericChartTableColumnConfig {
+    data: {
+        columnId: string,
+        labelColumn?: string
+        hierarchyColumn?: string
```

Comment:
No, since you still see the main data value even when you have a hierarchy view, as for the area (area name shown in larger text above hierarchy in smaller). So you could still have a label column for the main data value, though we're not doing that for `area_name`.

Reply:
Makes sense - thanks